src/base/bittorrent/session.cpp

```diff
@@ -1788,6 +1788,7 @@ void Session::processShareLimits()
                             torrent->pause();
                             logger->addMessage(tr("'%1' reached the maximum ratio you set. Paused.").arg(torrent->name()));
                         }
+                        continue; // Avoid possible crash in "torrent->name()" when processing seeding time
```

Comment:
Can you move the comment to commit message?

Reply:
Sure.

Reply:
IMO, it would be better to have something more meaningful as commit message body, e.g. "Avoid possible crash in "torrent->name()" when processing seeding time of torrent that already reached ratio limit."

Reply:
Ok